<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="gridFrame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>120</width>
      <height>320</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="details">
       <property name="text">
        <string>Aircraft Details</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="transferButton">
       <property name="text">
        <string>Transfer</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="type">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QProgressBar" name="fuelLevel">
       <property name="value">
        <number>42</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="serialNumber">
       <property name="text">
        <string>Serial Number</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="squadron">
       <property name="text">
        <string>Squadron</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>670</width>
      <height>430</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="fdTab">
     <attribute name="title">
      <string>Flight Deck</string>
     </attribute>
     <widget class="QGraphicsView" name="fdView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>640</width>
        <height>285</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="hangerTab">
     <attribute name="title">
      <string>Hanger</string>
     </attribute>
     <widget class="QGraphicsView" name="hangerView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>560</width>
        <height>190</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="missionTab">
     <attribute name="title">
      <string>On Mission</string>
     </attribute>
     <widget class="QListWidget" name="missionList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>210</width>
        <height>240</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="transitTab">
     <attribute name="title">
      <string>In Transit</string>
     </attribute>
     <widget class="QListWidget" name="transitList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>210</width>
        <height>240</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionaddAircraft"/>
    <addaction name="actioneditAircraft"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionaddAircraft"/>
   <addaction name="actioneditAircraft"/>
   <addaction name="actionremoveAircraft"/>
   <addaction name="actionrotateAircraft"/>
   <addaction name="actionviewSquadrons"/>
   <addaction name="actionArrived"/>
  </widget>
  <action name="actionaddAircraft">
   <property name="text">
    <string>Add Aircraft</string>
   </property>
  </action>
  <action name="actionviewSquadrons">
   <property name="text">
    <string>View Squadrons</string>
   </property>
  </action>
  <action name="actioneditAircraft">
   <property name="text">
    <string>Edit Aircraft</string>
   </property>
  </action>
  <action name="actionremoveAircraft">
   <property name="text">
    <string>Remove Aircraft</string>
   </property>
  </action>
  <action name="actionrotateAircraft">
   <property name="text">
    <string>Rotate Aircraft</string>
   </property>
  </action>
  <action name="actionArrived">
   <property name="text">
    <string>Arrived</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionaddAircraft</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addAircraft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transferButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>transferAircraft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionviewSquadrons</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>viewSquadrons()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actioneditAircraft</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>editAircraft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionremoveAircraft</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeAircraft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionrotateAircraft</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>rotateAircraft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>missionList</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>setActiveMission()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>786</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transitList</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>setActiveTransit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>787</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionArrived</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>arrived()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addAircraft()</slot>
  <slot>transferAircraft()</slot>
  <slot>setActiveHanger()</slot>
  <slot>setActiveTransit()</slot>
  <slot>setActiveMission()</slot>
  <slot>viewSquadrons()</slot>
  <slot>editAircraft()</slot>
  <slot>removeAircraft()</slot>
  <slot>rotateAircraft()</slot>
  <slot>setActiveFD()</slot>
  <slot>arrived()</slot>
 </slots>
</ui>
